archive.sh

```diff
@@ -24,7 +45,16 @@ composer install --no-dev
 cd /tmp
 
 # Run archive command to create the zip in the root directory
-zip -r9 ${ROOT_DIR}/FinSearchUnified-${VERSION}.zip FinSearchUnified -x FinSearchUnified/phpunit.xml.dist FinSearchUnified/shopware-phpcs.xml FinSearchUnified/Tests/\*
+zip -r9 ${ROOT_DIR}/FinSearchUnified-${VERSION}.zip FinSearchUnified -x FinSearchUnified/phpunit.xml.dist
+FinSearchUnified/shopware-phpcs.xml FinSearchUnified/Tests/\*
```

Comment:
This doesn't work. `shopware-phpcs.xml` and the `Tests` folder are in the archive now. I guess you will have to do something like
```
zip -r9 ${ROOT_DIR}/FinSearchUnified-${VERSION}.zip FinSearchUnified -x FinSearchUnified/phpunit.xml.dist \
FinSearchUnified/shopware-phpcs.xml FinSearchUnified/Tests/\*
```